blueflood-core/src/main/java/com/rackspacecloud/blueflood/outputs/formats/ErrorResponse.java

```diff
@@ -31,24 +31,30 @@ public String toString() {
     public static class ErrorData {
 
         private String tenantId;
+        private Long timestamp;
         private String metricName;
         private String source;
         private String message;
 
         public ErrorData() {
         }
 
-        public ErrorData(String tenantId, String metricName, String source, String message) {
+        public ErrorData(String tenantId, String metricName, String source, String message, Long timestamp) {
```

Comment:
Maybe more comment on what this timestamp represent? Is this ingest time? Collection time?